oxidation/libparsec/crates/core_fs/Cargo.toml

```diff
@@ -15,13 +15,18 @@ libparsec_client_types = { path = "../client_types" }
 # Disable Diesel's default feature to remove `32-column-tables` which slowdown compilation
 # quiet a lot (Diesel crate compilation on my machine: 19s with vs 8s without).
 # Note this is fine as long as we use tables with at most 16 columns.
-diesel = { version = "2.0.2", features = ["sqlite", "r2d2", "with-deprecated"], default-features = false }
+diesel = { version = "2.0.2", features = ["sqlite"], default-features = false }
```

Comment:
`with-deprecated` doesn't accelerate build ?

Reply:
Why would it should ?

Reply:
I don't remember why we add this feature

Reply:
Nether do I :shrug:

Reply:
@touilleMan https://github.com/Scille/parsec-cloud/pull/3759

Reply:
`with-deprecated` or `default-features=false` which speed up ?